vector/src/main/java/im/vector/app/features/pin/lockscreen/ui/LockScreenListener.kt

```diff
@@ -19,45 +19,45 @@ package im.vector.app.features.pin.lockscreen.ui
 /**
  * Listener class to be notified of any event that could happen in the lock screen UI.
  */
-open class LockScreenListener {
+interface LockScreenListener {
     /**
      * In PIN creation mode, called when the first PIN code has been entered.
      */
-    open fun onFirstCodeEntered() {}
+    fun onFirstCodeEntered() {}
```

Comment:
The default impl could just return Unit but it's a detail
`fun onFirstCodeEntered() = Unit`